src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -795,6 +807,24 @@ public function theCustomerCompletedTheOrder(): void
         $this->objectManager->flush();
     }
 
+    /**
+     * @Given /^there is a (customer "[^"]+") that has placed 2 orders with numbers "([^"]*)" and "([^"]*)"$/
+     */
+    public function thereIsCustomerThatPlacedTwoOrders(
+        CustomerInterface $customer,
```

Comment:
Transformer problem - it's taking a string

Reply:
You need to use `sylius.behat.context.transform.customer` in the suite 🖖

Reply:
If I see correctly, you didn't use `sylius.behat.context.transform.customer` in your suite